src/components/bookmarksList/bookmarksList.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import AccountVisual from '../accountVisual';
+import EmptyState from '../emptyStateV2';
+import Box from '../boxV2';
+import { tokenMap } from '../../constants/tokens';
+import regex from '../../utils/regex';
+import svg from '../../utils/svgIcons';
+import { SecondaryButtonV2 } from '../toolbox/buttons/button';
+import routes from '../../constants/routes';
+import styles from './bookmarksList.css';
+
+class BookmarksList extends React.Component {
+  onBookmarkSelected(address) {
+    const { history } = this.props;
+    history.push(`${routes.explorer.path}/accounts/${address}`);
+  }
+
+  getBookmarkListBasedOnSelectedToken() {
+    const { bookmarks, token } = this.props;
+
+    const actualBookmarks = { ...bookmarks };
+    return actualBookmarks[token.active].length > 5
```

Comment:
Don't need to check the length of the bookmarks here, you can just `return bookmarks[token.active].slice(0, 5);`